homeassistant/components/sensor/geo_rss_events.py

```diff
@@ -0,0 +1,253 @@
+"""
+Generic GeoRSS events service.
+
+Retrieves current events (typically incidents or alerts) in GeoRSS format, and
+shows information on events filtered by distance to the HA instance's location
+and grouped by category.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/sensor.geo_rss_events/
+"""
+
+import logging
+from datetime import timedelta
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.components.sensor import PLATFORM_SCHEMA, ENTITY_ID_FORMAT
+from homeassistant.const import (STATE_UNKNOWN, CONF_SCAN_INTERVAL,
+                                 CONF_UNIT_OF_MEASUREMENT, CONF_NAME)
+from homeassistant.helpers.entity import Entity, generate_entity_id
+from homeassistant.util import Throttle
+
+REQUIREMENTS = ['feedparser==5.2.1', 'haversine==0.4.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_CATEGORY = 'category'
+ATTR_DISTANCE = 'distance'
+ATTR_TITLE = 'title'
+
+CONF_CATEGORIES = 'categories'
+CONF_RADIUS = 'radius'
+CONF_URL = 'url'
+
+DEFAULT_ICON = 'mdi:alert'
+DEFAULT_NAME = "Event Service"
+DEFAULT_RADIUS_IN_KM = 20.0
+DEFAULT_SCAN_INTERVAL = timedelta(minutes=5)
+DEFAULT_UNIT_OF_MEASUREMENT = 'Events'
+
+DOMAIN = 'geo_rss_events'
+
+# Minimum time between updates from the source.
+MIN_TIME_BETWEEN_UPDATES = timedelta(minutes=1)
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Required(CONF_URL): cv.string,
+    vol.Optional(CONF_RADIUS, default=DEFAULT_RADIUS_IN_KM): vol.Coerce(float),
+    vol.Optional(CONF_NAME, default=None): cv.string,
+    vol.Optional(CONF_CATEGORIES, default=[]): vol.All(cv.ensure_list,
+                                                       [cv.string]),
+    vol.Optional(CONF_UNIT_OF_MEASUREMENT,
+                 default=DEFAULT_UNIT_OF_MEASUREMENT): cv.string,
+    vol.Optional(CONF_SCAN_INTERVAL,
```

Comment:
Remove this, since you should have defined `SCAN_INTERVAL` at the module level instead. `CONF_SCAN_INTERVAL` is already part of the base `PLATFORM_SCHEMA` so it can be overridden by the user.